File: py/leetcode/lc1342_test.py
import unittest
import leetcode.lc1342 as lc


class Test(unittest.TestCase):
    def setUp(self):
        self.solution = lc.Solution()

    def test(self):
        self._test(14, 6)
        self._test(8, 4)
        self._test(123, 12)

    def _test(self, num, expected):
        self.assertEqual(expected, self.solution.numberOfSteps(num))

File: go/src/leetcode/lc1342_test.go
package leetcode

import "testing"

func Test_numberOfSteps(t *testing.T) {
	test1342(14, 6, t)
	test1342(8, 4, t)
	test1342(123, 12, t)
}

func test1342(num int, expected int, t *testing.T) {
	if expected != numberOfSteps(num) {
		t.Error()
	}
}
